runtime/gc_ctrl.c

```diff
@@ -655,6 +673,7 @@ void caml_init_gc (uintnat minor_size, uintnat major_size,
   caml_percent_max = norm_pmax (percent_m);
   caml_init_major_heap (major_bsize);
   caml_major_window = norm_window (window);
+  CAML_EVENTLOG_INIT();
```

Comment:
Why did you move this into the middle of the GC initialization?

Reply:
It was actually meant to move at the end of the GC initialization. Thank you.
Maybe it is still not quite the best position, maybe it should be moved to the runtime startup code, after GC's initialization ?

Reply:
Yes, that would make sense. Eventlog is not part of the GC and we may want to extend it to other parts of the runtime in the future.